### ISSUE TITLE
replace all sklearn -> scikit-learn

### ISSUE THREAD

**Issue by casperdcl** [iterative/cml.dev#330]
Quite a few instances in https://github.com/search?l=Text&q=org%3Aiterative+sklearn&type=Code

See e.g. https://github.com/iterative/example_cml/pull/256, https://github.com/iterative/cml_dvc_case/pull/23

**Comment by tasdomas** [comment]
Is there really a supply chain issue here? `sklearn` is a dummy package that specifies `scikit-learn` as its only dependency. Sure, it's just a workaround for the import and package name mismatch in `scikit-learn`, but apart from the additional `sklearn==0.0` line in the output of `pip freeze` there's no significant issue.

**Comment by dacbd** [comment]
shot in the dark, but there have been a few issues of users asking for support on discord that has stemmed from pip install issues and sk[scikit-]learn versions, so if this is related to that :+1:

**Comment by casperdcl** [comment]
yes, we should fix it since it looks like the dummy package doesn't always install properly
